### ISSUE TITLE
Teams PS: Get-CsOnlineUser does not show the Online Dialout Policy

### ISSUE THREAD

**Issue by donlassini** [MicrosoftDocs/office-docs-powershell#8990]
First I run this using Teams Online PS 3.1.1:
```
PS> $UPN = "<Teams and Voice enabled user's UPN>"
PS> Grant-CsDialOutPolicy -Identity $UPN -PolicyName "tag:DialoutCPCandPSTNDomestic"
```

The I run
```
PS> Get-CsOnlineUser $UPN | fl OnlineDialOutPolicy, WhenChanged
```
For Teams Online PS 3.1.1 and 3.2.0, I get
```
OnlineDialOutPolicy :
WhenChanged         : 22-02-2022 08:18:40

```
If I run the command in Teams Online PS 2.3.1, I get a result:
```
OnlineDialOutPolicy : DialoutCPCandPSTNDomestic
WhenChanged         : 22-02-2022 09:18:00

```
So the OnlineDialOutPolicy is not shown in Teams PS 3.1.1 and later.
I have not tested 3.0 releases prior to 3.1.1.

(Never mind that the WhenChanged attribute is UTC for the newer modules.)
Also, it's not e.g. a replication issue since I would still not get a result in 3.1.1 even after testing 2.3.1.

**Comment by donlassini** [comment]
Closed; it's the same as #8935

**Comment by olanga** [comment]
Just a remark. 
Also OnPremLineURI value returns empty  with Get-CsOnlineUser $UPN
Workaround: Get-CsOnlineUser -filter "UserPrincipalName -eq '$UPN'"
